content/learn/book/assets/loading-assets/_index.md

```diff
@@ -4,6 +4,7 @@ weight = 1
 sort_by = "weight"
 template = "book-section.html"
 page_template = "book-section.html"
+insert_anchor_links = "right"
```

Comment:
Yeah, this would be a rebasing issue... Let me check if they're duplicated.

Reply:
Nope, not duplicated. IDK why this is showing up on Github; let me try rebasing again.

Reply:
Alright, well that's fine then, it doesn't matter that much, I was just a bit surprised to see that.

Reply:
It was added in #280. Every single `_index.md` file needs this so that it is consistent throughout the website. Therefore whenever you add a new `_index.md` file you would have to add `insert_anchor_links` at the top of the file. Not sure why there are files highlighted as changed here even though they already have this lint in the `new_book` branch.

Reply:
Yeah, that was my confusion. That PR is already merged, so I don't get why git still shows it as changed. As long as the end result is that we have mdlint _and_ anchors everywhere I'm happy.